### PR TITLE
Feature function specs

### PR DESCRIPTION
**Pull request by faloi**
Closes uqbar-project/mumuki-gobstones-server#31

### CODE REVIEW

File: lib/stones-spec/postcondition.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    def initialize(return_value)
+      @return_value = return_value.to_s
+    end
+
+    def validate(_initial_board_file, _actual_final_board_gbb, actual_return)
+      normalized_actual_return = actual_return.strip
+
+      if normalized_actual_return == return_value
+        ['', :passed]
+      else
```
Comment (flbulgarelli) [inline lib/stones-spec/postcondition.rb:80]: What should we show here?

Comment (faloi) [inline lib/stones-spec/postcondition.rb:80]: Could be a tuple (board, function with parameters, result)?

Something like:

```
myFunction(1, Norte) = 25
         0         1        
    +---------+---------+   
    |         |  1N     |   
  1 |         |         | 1 
    |         |         |   
    +XXXXXXXXX+---------+   
    X      2A X         |   
  0 X         X         | 0 
    X      1R X      2R |   
    +XXXXXXXXX+---------+   
         0         1        
```

Comment (flbulgarelli) [inline lib/stones-spec/postcondition.rb:80]: Yes, that seems ok for me. Do you want to implement it?

Comment (faloi) [inline lib/stones-spec/postcondition.rb:80]: Later, along with the failed output improvement: https://github.com/uqbar-project/stones-spec/issues/16

Comment (flbulgarelli) [inline lib/stones-spec/postcondition.rb:80]: Good

**Pull request merged**